Декоративные элементы всех страниц

NataliZin85 opened the pull request:
---
:mortar_board: [Декоративные элементы всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2050153/tasks/10)

NataliZin85 committed:
buttons-changed

NataliZin85 committed:
header-icons

NataliZin85 committed:
advantages - icons

NataliZin85 committed:
footer-icons

NataliZin85 committed:
breadcrumbs

NataliZin85 committed:
class select-control

NataliZin85 committed:
product-view-product-item

NataliZin85 committed:
продолжаю работать над катaлогом и исправила header

NataliZin85 committed:
переделала футер

NataliZin85 committed:
navigation-user-link

NataliZin85 committed:
header-псевдоклассы

NataliZin85 committed:
main-псевдоклассы кнопок

NataliZin85 committed:
footer-псевдоэлементы

то что закоментированно, не работает. надо разобраться почему

amaringo commented on catalog.html:
```diff
@@ -27,24 +27,24 @@
           </li>
           <li class="navigation-user-item">
-            <a class="navigation-user-link-favorite" href="#">
+            <a class="navigation-user-link navigation-user-link-favorite" href="#">
```
Осталось убрать подчёркивание текста в ссылке
![image](https://user-images.githubusercontent.com/11677622/178176164-4766b294-46e3-4e0b-8114-a7a964fe5355.png)

amaringo commented on catalog.html:
```diff
@@ -27,24 +27,24 @@
       <nav class="navigation-user">
```
![image](https://user-images.githubusercontent.com/11677622/178176387-8215e1d8-8c5a-4065-aaea-0afc8b9b9792.png)
Опустить на 5 пикселей ниже, добавить z-index

amaringo replied:
точнее, высоту хедера надо убавить на 5, сейчас он слишком высокий

amaringo commented on index.html:
```diff
@@ -143,17 +143,17 @@ <h2 class="subscription-title">Подпишитесь на рассылку</h2>
         <ul class="footer-social-list">
           <li class="footer-social-item">
-            <a class="button-vk" href="https://vk.com/htmlacademy">
+            <a class="footer-social-button button-vk" href="https://vk.com/htmlacademy">
```
Нет состояний у footer-social-button.

NataliZin85 replied:
Пока нет, они не работают. То, что я писала, что не разобралась... закоментированно...

amaringo commented on index.html:
```diff
@@ -143,17 +143,17 @@ <h2 class="subscription-title">Подпишитесь на рассылку</h2>
+            <a class="footer-social-button button-youtube" href="https://www.youtube.com/user/htmlacademyru">
               <span class="visually-hidden">Наш Ютуб</span>
             </a>
           </li>
```
Нет состояний у footer-link-logo, тут можно только opacity менять.

NataliZin85 replied:
тоже самое, когда задаю другой цвет, то он окрашивает всю область или не работает...

NataliZin85 committed:
поправки+ верхушка каталога

NataliZin85 committed:
catalog-cards иконки и псевдосвойства

NataliZin85 committed:
pagination

NataliZin85 committed:
главная страница

поправила footer, вставила разделитель в hero, сделала всю страницу в пиксель-перфект

NataliZin85 committed:
переделала кнопки и поменяла футер на страницы каталога

NataliZin85 committed:
подправила цыфру 12 и поправила логотип в футере при наведении и нажатии

NataliZin85 committed:
страница каталога

пиксель перфект

NataliZin85 committed:
appearance: none
